tests/smoke/features/step_definitions/app.files.ts

```diff
@@ -178,3 +175,196 @@ When(
     }
   }
 )
+
+When(
+  '{string} creates new versions of the following file(s)',
+  async function (this: World, stepUser: string, stepTable: DataTable): Promise<void> {
+    const actor = this.actorContinent.get({ id: stepUser })
+    const { allFiles: allFilesPage } = new FilesPage({ actor })
+    const uploadInfo = stepTable.hashes().reduce((acc, stepRow) => {
+      const { to, resource } = stepRow
+
+      if (!acc[to]) {
+        acc[to] = []
+      }
+
+      acc[to].push(this.fileContinent.get({ name: resource }))
```

Comment:
```suggestion
      acc[to].push(this.fileContent.get({ name: resource }))
```

Maybe this is meant? I guess that the existing method is named `fileContinent`

Reply:
You're probably right, I am not sure because code is not mine. We use `ActorContinent` `userContinent` `fileContinent`. @fschade is it typo? change everywhere?

Reply:
continent is fine https://github.com/owncloud/web/blob/shareFile/tests/smoke/support/world/user.ts